kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorNetworkIT.java

```diff
@@ -285,9 +288,8 @@ private void emptyBucket(String bucketName) {
     // delete all objects to empty bucket
     ObjectListing objectListing = S3Client.listObjects(bucketName);
     while (true) {
```

Comment:
Here I meant if we could use the below instead of the whole first while loop, not within it. 
```
for (S3ObjectSummary s3ObjectSummary : objectListing.getObjectSummaries()) { 
    S3Client.deleteObject(bucketName, s3ObjectSummary.getKey());
}
```

Reply:
We are using the while loop to get the next batch of objects (if exist) after getting the initial object list. This is on the same lines as mentioned in the [above comment ](https://github.com/confluentinc/kafka-connect-storage-cloud/pull/373#discussion_r518889538)